code/modules/projectiles/ammunition/ballistic/rifle.dm

```diff
@@ -7,23 +7,51 @@
 	caliber = CALIBER_STRILKA310
 	projectile_type = /obj/projectile/bullet/strilka310
 
+	/// The corresponding .310 Short casing we can be cut down to. Cannot be cut down if null.
+	var/short_casing = /obj/item/ammo_casing/short310
+
 /obj/item/ammo_casing/strilka310/Initialize(mapload)
 	. = ..()
 
 	AddElement(/datum/element/caseless)
+	register_context()
+
+/obj/item/ammo_casing/strilka310/examine(mob/user)
+	. = ..()
+	if(short_casing)
+		. += span_notice("It can be <b>cut</b> to make a shortened casing.")
+
+/obj/item/ammo_casing/strilka310/add_context(atom/source, list/context, obj/item/held_item, mob/living/user)
+	if(istype(held_item) && short_casing && (held_item.tool_behaviour == TOOL_WIRECUTTER || held_item.get_sharpness()))
+		context[SCREENTIP_CONTEXT_LMB] = "Shorten casing"
```

Comment:
this doesn't specify well enough that it's a different ammo type for a different weapon. if the gun is going to remain hyper-rare it doesn't even make sense to make it a option imo.